<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人信息</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;
color:black;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color:rgb(60, 60, 60);
    font-size: 18px; /* 设置字体大小 */</string>
       </property>
       <property name="text">
        <string>专业</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="major_tag">
       <property name="styleSheet">
        <string notr="true">color:rgb(70, 70, 70);
    font-size: 16px; /* 设置字体大小 */</string>
       </property>
       <property name="text">
        <string>通用人工智能</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="major">
       <property name="styleSheet">
        <string notr="true">color:rgb(70, 70, 70);
    font-size: 16px; /* 设置字体大小 */</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color:rgb(60, 60, 60);
    font-size: 18px; /* 设置字体大小 */</string>
       </property>
       <property name="text">
        <string>年级</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="grade_tag">
       <property name="styleSheet">
        <string notr="true">color:rgb(70, 70, 70);
    font-size: 16px; /* 设置字体大小 */</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="grade">
       <property name="styleSheet">
        <string notr="true">color:rgb(70, 70, 70);
    font-size: 16px; /* 设置字体大小 */</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color:rgb(60, 60, 60);
    font-size: 18px; /* 设置字体大小 */</string>
       </property>
       <property name="text">
        <string>双学位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="double_major_tag">
       <property name="styleSheet">
        <string notr="true">color:rgb(70, 70, 70);
    font-size: 16px; /* 设置字体大小 */</string>
       </property>
       <property name="text">
        <string>无</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="double_major">
       <property name="styleSheet">
        <string notr="true">color:rgb(70, 70, 70);
    font-size: 16px; /* 设置字体大小 */</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="set_pinfo">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: rgb(240, 245, 255); /* 设置背景颜色 */
color:rgb(70, 70, 70);
    font-size: 16px; /* 设置字体大小 */
}
</string>
       </property>
       <property name="text">
        <string>更改个人信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
